spec/erb_lint/cli_spec.rb

```diff
@@ -141,7 +141,7 @@ def run(_processed_source)
         context 'when no errors are found' do
           let(:args) { ['--enable-linter', 'linter_without_errors', linted_file] }
 
-          it 'shows no that errors were found to stderr' do
+          it 'shows no that errors were found to stdout' do
```

Comment:
Clean up test description.

Reply:
I think it should also say `shows that no errors...`